cpp/examples/tutorial_examples/CMakeLists.txt

```diff
@@ -0,0 +1,20 @@
+project(MyExample)
+set(CMAKE_CXX_STANDARD 11)
```

Comment:
I didn't actually see it in the CMakeFiles -- I'm guessing it gets checked in CI, but not built by the user unless done very intentionally?

Reply:
Er, sorry, what are you referring to?

The other examples have a CMakeLists.txt that gets included by the main one; it uses Arrow-specific CMake config to make it build as part of CI

https://github.com/apache/arrow/blob/cdab96507561592e20253992b566ba729ebc7a3f/cpp/CMakeLists.txt#L1006-L1009

but I assume you want this to be a standalone CMakeLists.txt that doesn't rely on having all of Arrow's build system available

Reply:
Sorry, I meant that minimal_build isn't included in the CMakeLists.txt at a higher level.

Reply:
Yes, that's intentional, because it's meant to be a standalone example - instead there's a separate nightly CI job that exercises it (I'm not sure where exactly off the top of my head though)

Reply:
Ah

https://github.com/apache/arrow/blob/cdab96507561592e20253992b566ba729ebc7a3f/docker-compose.yml#L415-L434

and 

https://github.com/apache/arrow/blob/cdab96507561592e20253992b566ba729ebc7a3f/ci/docker/ubuntu-22.04-cpp-minimal.dockerfile

Reply:
Wait, maybe not that one

Reply:
Here? https://github.com/apache/arrow/blob/0ffcc5939ed9fc0a4ca23492d68d2df0661f2bf9/dev/tasks/tasks.yml#L1587-L1594

Reply:
(But probably don't need to worry about this yet…)

Reply:
For sure, we'll cross that bridge once everything's more stable.

Reply:
Remembered this, working on it today.